### PR TITLE
Use enode RPC in UI

### LINKED ISSUE #3106: Add JSAPI for enode lookup, display under status
Implemented in https://github.com/ethcore/parity/pull/3096

### PR DESCRIPTION
**Pull request by ngotchac**
Closes #3106 

Gets the enode address from the RPC call and display it.
Added it to JS API.

Needs #3096 to be merged.

### CODE REVIEW

**Comment by jacogr** [comment]
1. Merge in master first - the PR has been merged, so now shows Rust changes.
2. Missing addition of interface in jsonrpc

File: js/src/redux/providers/status.js [hunk 1]
```diff
@@ -68,11 +82,17 @@ export default class Status {
 
   _pollStatus = () => {
     const { secureToken, isConnected, isConnecting, needsToken } = this._api;
+
     const nextTimeout = (timeout = 1000) => {
       setTimeout(this._pollStatus, timeout);
     };
 
+    if (isConnected !== this._store.getState().nodeStatus.isConnected) {
```
Comment (derhuerst) [inline js/src/redux/providers/status.js]:
```js
const wasConnected = this._store.getState().nodeStatus.isConnected
if (isConnected !== wasConnected) …
```

**Comment by derhuerst** [comment]
I think monospaced text will be better to read in this case. Also, will the enode actually be relevant to the majority of the users? What about putting it in a tooltip on the "x/y/z peers" text?

**Comment by jacogr** [comment]
Sadly we don't use monospaced fonts on the status page atm, so it would completely out of place if added. On the status page we have all the technical status settings, which is why it is not visible to users. (e.g. miner setting, mining author, ports, etc.) So the positioning is correct.

Happy with what is there overall, right place, right positioning.

**Comment by gavofyork** [comment]
i'd like to get rid of long hex strings from the UI in general. this includes at least addresses and enode ids.

they're easily replaced by a copy button (in case the user actually wants them) - possibly mouse-over sensitive, and a rendering like `abcdef...abcd`

**Pull request merged**